src/potdata/sampler.py

```diff
@@ -175,10 +182,71 @@ def __init__(
         self.reachable_ratio = reachable_ratio
         self.core_ratio = core_ratio
 
-        self._indices: list[int] = None
+        self._indices: list[int] = []
 
         np.random.seed(seed)
 
+    def _get_soap_vectors(self, data: list[Structure]) -> list[np.ndarray]:
+        """Convert structures to SOAP vectors.
+           Take Li atom in Li3YCl6 as an example."""
```

Comment:
Should not hard code it, for example, the species should be obtained from the structures.

Reply:
Fixed